Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -643,7 +643,7 @@ WI.initializeTarget = function(target)
 
         // COMPATIBILITY (iOS 13.4): Page.setShowRulers was removed.
         if (target.hasCommand("Page.setShowRulers") && WI.settings.showRulers.value)
-            target.PageAgent.setShowRulers(true);
+            target.PageAgent.setShowRulers(true).catch((error) => { console.error(error); }); // FIXME <rdar://105244623>
```

Comment:
Style Nit: with the comment, I'd just format this as a multi-line statement
```
target.PageAgent.setShowRulers(true).catch((error) => {
    // FIXME <rdar://105244623> Explanation words and stuff.
    console.error(error);
});
```

Also, can we do a bit of exposition after the radar number, e.g. "iOS platforms without a matching legacy protocol definition will fall back to the local macOS protocol definition."